IdentityCore/src/requests/MSIDInteractiveTokenRequest.m

```diff
@@ -229,18 +230,18 @@ - (void)acquireTokenWithCode:(NSString *)authCode
                                                                                                            codeVerifier:self.webViewConfiguration.pkce.codeVerifier
                                                                                                                authCode:authCode
                                                                                                           homeAccountId:self.authCodeClientInfo.accountIdentifier];
+    tokenRequest.responseSerializer = [[MSIDAADTokenResponseSerializer alloc] initWithOauth2Factory:self.oauthFactory];
```

Comment:
what if it's not AAD? Should we maybe set token response serializer in factory itself where MSIDAuthorizationCodeGrantRequest is created?

Reply:
Agree, changed.